### PR TITLE
CC-40 Investigate SVG image implementation

### PR DESCRIPTION
**Pull request by PavelPyasecky**
[CC-40](https://pavelpiasetski.atlassian.net/browse/CC-40)

### CODE REVIEW

**Comment by 17Sergey** [comment]
Great job! I like the idea of creating SvgContainer for no-repeating svg code. Commented one file that's better to fix

File: src/components/common/SvgContainer/SvgContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+type SvgContainerProps = {
+    path: string,
+    color: string
+};
+
+export const SvgContainer: FunctionComponent<SvgContainerProps> = ({path, color}) => {
+    return (
+            <SvgLoader path={path}>
+                <SvgProxy selector="rect" fill={color} />
+                <SvgProxy selector="ellipse" fill={color} />
```
Comment (17Sergey) [inline src/components/common/SvgContainer/SvgContainer.tsx]: In our svg there is no `ellipse` tag. So we don't need it.
image

**Review by 17Sergey — CHANGES_REQUESTED** [review]

**Comment by PavelPyasecky** [comment]
@17Sergey After rereading your comment I've understood that making SvgContainer component as common was not really good idea. I've moved it to the NewsBlock.

**Review by 17Sergey — APPROVED** [review]

**Comment by 17Sergey** [comment]
Great job! Everything is ready

**Pull request merged**